test/docker/entrypoint.sh

```diff
@@ -13,6 +13,17 @@ oldpwd=$(pwd)
 cp -a . /work
 cd /work
 
+# Dependencies for running sshd and connecting to it
+ssh-keygen -A
+mkdir -p ~/.ssh/ /var/run/sshd
+echo "localhost $(cat /etc/ssh/ssh_host_ed25519_key.pub)" >>~/.ssh/known_hosts
+ssh-keygen -t ed25519 -N '' -f "$HOME/.ssh/id_ed25519"
```

Comment:
At least on alpine:
> `test/docker/entrypoint.sh: line 17: ssh-keygen: not found`

https://github.com/scop/bash-completion/actions/runs/14454159615/job/40533467548?pr=1367#step:4:46

Reply:
Nevermind, falls in the noted "CI in this PR fails because the images are only pushed after merging" category I suppose, openssh-server does pull in openssh-keygen which has ssh-keygen.